GraphBLAS/Source/Template/GB_atomics.h

```diff
@@ -100,9 +100,10 @@
 // is no need for atomic reads/writes when compiling GraphBLAS on Windows
 // with MS Visual Studio.
 
-#if GBX86
+#if GBX86 || ! defined (_OPENMP) ||  (_OPENMP <= 201107 )
```

Comment:
That should be (_OPENMP < 201107) I think.

Reply:
Oops. My bad.
Should be fixed now.